java/memory/memory-core/src/test/java/org/apache/arrow/memory/TestOpens.java

```diff
@@ -20,11 +20,13 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class TestOpens {
   /** Instantiating the RootAllocator should poke MemoryUtil and fail. */
   @Test
+  @Ignore
```

Comment:
why ignored?

Reply:
This has been cherry-picked from [here](https://github.com/sirensolutions/arrow/commit/9735b9928b9b4da540424d6d149509a80050f158#diff-3f85a254c9c4b8dc77a549a649c569a1439e4b356a9879e71079852927ea1bb0R29). I think this was ignored as the tests fails on JDK 16+ (based on [comment](https://github.com/sirensolutions/arrow/commit/9735b9928b9b4da540424d6d149509a80050f158#diff-3f85a254c9c4b8dc77a549a649c569a1439e4b356a9879e71079852927ea1bb0R31))

Reply:
ok